pkgs/applications/qubes/qubes-vmm-xen/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  fetchpatch,
+  callPackage,
+  ocaml-ng,
+  xen,
+  ...
+}@genericDefinition:
+
+let
+  pname = "qubes-vmm-xen";
+  branch = "4.19";
+  version = "4.19.0";
+  revision = "3";
+  latest = true;
+
+  xenPatches = import ../../virtualization/xen/generic/patches.nix {
+    inherit lib fetchpatch;
+  };
+  xenPatchList = lib.lists.flatten (
+    with xenPatches;
+    [
+      XSA_460
+      XSA_461
+    ]
+  );
+
+  qubesPatches = import ./patches.nix {
+    inherit fetchpatch version revision;
+  };
+  qubesPatchList = lib.lists.flatten (
+    with qubesPatches;
+    [
+      EFI_WORKAROUNDS
+      BACKPORTS
+      UPSTREAMABLE_PATCHES
+      QUBES_SPECIFIC_PATCHES
+      OTHERS
+    ]
+  );
+in
+
+callPackage
+  (import ../../virtualization/xen/generic/default.nix {
+    inherit
+      pname
+      branch
+      version
+      latest
+      ;
+
+    pkg.xen = {
+      rev = "026c9fa29716b0ff0f8b7c687908e71ba29cf239";
+      hash = "sha256-Q6x+2fZ4ITBz6sKICI0NHGx773Rc919cl+wzI89UY+Q=";
+      patches = xenPatchList ++ qubesPatchList;
+    };
+
+    meta = {
+      inherit branch;
+      description = "Qubes component: vmm-xen";
+      longDescription = ''
+        Qubes OS' modified version of the Xen Hypervisor.
+        Contains dozens of Qubes-specific patches, and is intended to power a Qubes OS Domain 0.
+
+        This is effectively a `xen-slim` build. Use with `qemu_qubes`.
+      '';
+      homepage = "https://qubes-os.org";
+      downloadPage = "https://github.com/QubesOS/qubes-vmm-xen";
+      license = xen.meta.license;
+      mainProgram = "xl";
+      platforms = lib.lists.intersectLists lib.platforms.linux lib.platforms.x86_64;
+      maintainers = with lib.maintainers; [ sigmasquadron ];
```

Comment:
I'm interested in maintaining QubesOS packages, please add me to maintainers (only qubes-vmm-xen, not xen itself)